### PR TITLE
Prevent migration of a Cloud Firewall-supported Linode to a non-CF Data Center

### PR DESCRIPTION
**Pull request by WilkinsKa1**
## Description

Adding Cloud Firewall to Region capabilities. To test, attempt migrating a Linode with Firewalls to a DC that doesn't have Firewalls as part of its capability list. Upon submit, you should receive an API error explaining to choose another DC that supports Firewalls.

I originally wanted  to have a way to prevent the user from getting into an error state so they can only select from regions with FW support, but the logic required to do this was tricky/expensive with the current way we would need to cross-reference attached Linodes for any particular Firewall. There are plans to add associated Firewalls to the Linode details response, but that hasn't happened yet.

## Type of Change
- Non breaking change ('update')

### CODE REVIEW

File: packages/manager/cypress/integration/firewall/migrate-linode-with-firewall.spec.ts [hunk 1]
```diff
@@ -132,9 +132,9 @@ describe('Migrate Linode With Firewall', () => {
     cy.findByText('Accept').click();
```
Comment (acourdavault) [inline packages/manager/cypress/integration/firewall/migrate-linode-with-firewall.spec.ts:132]: i think this test should be changed to test that we get the API Error and display it as we do not hide the DC

Comment (ndukatz) [reply packages/manager/cypress/integration/firewall/migrate-linode-with-firewall.spec.ts:141]: I added a commit to this pr with an e2e test that should cover this case.

**Review by Jskobos — APPROVED** [review]

**Review by acourdavault — APPROVED** [review]
lgtm, needs to have e2e fix for the click on the wrong item https://www.loom.com/share/4227983b8b2a4ebab82f315fcc9c2227

**Comment by ndukatz** [comment]
> lgtm, needs to have e2e fix for the click on the wrong item https://www.loom.com/share/4227983b8b2a4ebab82f315fcc9c2227

is anyone else seeing this behavior? I can't reproduce it so it's hard to fix

**Review by ndukatz — APPROVED** [review]
If no one else is seeing the issue with the test, we should probably just get this merged, then I can create a separate pr to fix the test if necessary.

**Pull request merged**